docs/grow-then-shrink-upgrade.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Grow-then-Shrink Upgrade
+displayed_sidebar: docsSidebar
```

Comment:
It shouldn’t be necessary to specify the `displayed_sidebar` because the document is listed in the sidebar already.